### DISCUSSION TITLE
Feature Request: Disable select core extensions without disabling all of them

### DISCUSSION THREAD

**Discussion by Nantris** [ueberdosis/tiptap#5538] [Feature Requests]
### Description

Some of the core extensions may need to be overridden by users, but I think the only way that's supported currently is to disable ALL core extensions.

### Use Case

I'd like to disable `ClipboardTextSerializer` but there doesn't seem to be any easy approach.

### Type

Other

**Comment by nperez0111** [comment]
We just introduced this feature. It has not yet been released https://github.com/ueberdosis/tiptap/pull/5514

But what is wrong with the clipboard serializer? Can we just add configuration for it

**Reply by Nantris** [reply]
That's awesome!

There's nothing wrong with it - this is just the second time I've found myself fighting against the core plugins. They're great for 99.99% of use cases, but that 0.01% of cases you find yourself fighting them to implement customizations can be a pain.
